WTWSMS/common/scripted_triggers/ACR_scripted_triggers.txt

```diff
@@ -83,6 +88,7 @@ is_ancient_religion_trigger = {
 		religion = hellenic_pagan
 		religion = egyptian_pagan
 		religion = aten_pagan
+		religion = assyrian_pagan
```

Comment:
It doesn't break anything. All aspects of it are intact, but Assyrian Paganism is also added. It is already being used, and I included Assyrian Paganism because it is _older_ than Hellenic Paganism, IIRC!